src/client/containers/login-page/login-page.component.js

```diff
@@ -14,7 +14,7 @@ function LoginPage() {
   const user = useContext(FirebaseContext);
 
   useEffect(() => {
-    if (user) history.replace('/dashboard');
+    if (user) history.replace('/list-batches');
```

Comment:
What is this about?

Reply:
Sorry, didn't realize that the list batches page comes up before so you can select which dashboard.

Currently you are redirected to this page even if you're not logged in though. That needs to be resolved.

Also the menu is completely blank. I think there should be options to log out and add batches (add batches, only provided you have authorization to see that).

Reply:
Probably involves changing the sidebar component a bit to allow for easier turning on/off options.